flow/scripts/generate_abstract.tcl

```diff
@@ -21,7 +21,15 @@ if {$design_stage >= 4} {
 # write_timing_model includes the source latency in the model
 set_clock_latency -source 0 [all_clocks]
 puts "Generating abstract views"
-log_cmd write_timing_model $::env(RESULTS_DIR)/$::env(DESIGN_NAME).lib
+if {[env_var_exists_and_non_empty CORNERS]} {
+  # corners
+  foreach corner $::env(CORNERS) {
+    log_cmd write_timing_model -corner $corner $::env(RESULTS_DIR)/$::env(DESIGN_NAME)_$corner.lib
+  }
+  unset corner
+} else {
+  log_cmd write_timing_model $::env(RESULTS_DIR)/$::env(DESIGN_NAME)_typ.lib
```

Comment:
is the change in .lib file intentional? It will break all users, no?

Reply:
This is a breaking change for users of ORFS, such as bazel-orfs. The output name is part of the ORFS interface/contract.